app/controllers/application_controller.rb

```diff
@@ -36,7 +38,8 @@ def routing_error
   private
 
   rescue_from 'Exception' do |exception|
-    log_error(exception)
+    # report the original 'cause' of the exception when present
+    log_exception_to_sentry(exception) unless SKIP_SENTRY_EXCEPTION_TYPES.include?(exception.class)
```

Comment:
Good catch.  I periodically look in Sentry to _find_ problems, even when things are working fine.  OTOH, I only look in the rails log for forensic purposes to solve a _known problem_.  I think it's worth not losing that forensic data, so I'll add some logic:

```ruby
if SKIP_SENTRY_EXCEPTION_TYPES.include?(exception.class)
  log_exception_to_sentry(exception)
else
  Rails.logger.error "#{exception.message}."
  Rails.logger.error exception.backtrace.join("\n") unless exception.backtrace.nil?
end
```